src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsController.java

```diff
@@ -120,7 +135,10 @@
     }
 
     private boolean isSolicitorAgreedToReceiveEmails(Map<String, Object> mapOfCaseData, String solicitorAgreeToReceiveEmails) {
-        return YES_VALUE.equalsIgnoreCase(Objects.toString(mapOfCaseData
-                .get(solicitorAgreeToReceiveEmails)));
+        return YES_VALUE.equalsIgnoreCase(Objects.toString(mapOfCaseData.get(solicitorAgreeToReceiveEmails)));
+    }
+
+    private boolean isPaperApplication(Map<String, Object> mapOfCaseData) {
+        return YES_VALUE.equalsIgnoreCase(Objects.toString(mapOfCaseData.get(PAPER_APPLICATION)));
```

Comment:
looks like we could create a common method: `private boolean isTrue(Map<String, Object> mapOfCaseData, String key) {
        return YES_VALUE.equalsIgnoreCase(Objects.toString(mapOfCaseData.get(key)));
    }`

then update lines 138 and 142 to use this (possible reusable across other classes?)